integration-tests/bats/sql-local-remote.bats

```diff
@@ -108,3 +121,72 @@ teardown() {
     [[ "$output" =~ "defaultDB does not exist" ]] || false
 }
 
+@test "sql-local-remote: test 'status' and switch between server/no server" {
+    start_sql_server defaultDB
+
+    run dolt --user dolt status
+    [ "$status" -eq 0 ] || false
+    [[ "$output" =~ "On branch main" ]] || false
```

Comment:
I think a test which would be easier to read and more correct would be to take all of the output from the remote invocation, and compare it with all the output of the local invocation. The point of the test is to verify that the two modes produce the same output, not to make sure it reads like status

Reply:
Adding the comparison between local and remote executions.

Reply:
You can go further and drop all of the line - line validation. Up to you.

Reply:
I think there's value to doing at least one check of the contents, so I'll remove the repeat checks.